api/src/main/resources/liquibase.xml

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- <databaseChangeLog
+		xmlns="http://www.liquibase.org/xml/ns/dbchangelog/2.0"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/2.0
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd"> -->
+
+<databaseChangeLog logicalFilePath="liquibase-update-to-latest.xml" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
```

Comment:
We do not need logicalFilePath for modules.

Reply:
okay. I'll remove that

Reply:
Have you also seen my comment on JIRA?

Reply:
I have. Just responded